client-rpc/src/bookmap/bookmap.py

```diff
@@ -612,6 +617,19 @@ def resize_order(addon: typing.Dict[str, object],
         _stop_addon()
 
 
+def subscribe_to_indicator(addon: typing.Dict[str, object], addon_name: str, alias: str) -> typing.NoReturn:
```

Comment:
[-> typing.NoReturn](https://docs.python.org/3/library/typing.html#typing.NoReturn) means that the function will _never_ return. Instead, if there's nothing to return, we should do `-> None`.